.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: csharp
 mono:
-- 5.2.0
+- 3.12.1 # Csharp 5.0
```

Comment:
what are u doing Fahad when .net framework 5.0 released? I am not interested in csharp # 5.0 or 6.0 or 7.0...